src/components/testimonialGrid/TestimonialGrid.js

```diff
@@ -0,0 +1,37 @@
+import TestimonialCard from '../testimonialCard/TestimonialCard'
+import './testimonial-grid.css'
+
+function TestimonialGrid(props) {
+  return (
+    <div className='testimonial-grid grid'>
```

Comment:
I feel like the grid implementation could be simplified, using less nesting and less util classes.

Reply:
You should try to keep the html structure flat (no nesting) and then possibly use `grid-template-areas` to name the areas and adjust their layout accordingly.